Frontend/Services/ProductService.cs

```diff
@@ -26,5 +26,11 @@ public async Task<Product> GetProductById(int id)
         {
             return await httpClient.GetJsonAsync<Product>($"api/v1.0/products/{id}");
         }
+        public async Task<Product> AddProducts(Product product)
+        {
+            //product.Id = null;
+            return await httpClient.PostJsonAsync<Product>("api/v1.0/products", product);
+        }
+
```

Comment:
Från vilken httpaddres hämtas detta? Hämtas det från localhost Backend API? Det enda jag kan läsa ut är en lokal genväg: ($"api/v1.0/products/{id}");

Du får tänka på nu att vår backend ligger i Azure. Därför fungerar inte en lokal sökväg längre. Så när jag kör Developbranchen nu, så fungerar den inte, utan klagar på dina rader jag markerat alt. senare får vi hitta en lösning med en eller constant variabel som alla kommer åt som tar in strängen: https://g5store.azurewebsites.net.

Troligtvis behöver det stå något i stil med:
return await httpClient.PostJsonAsync<Product>("https://g5store.azurewebsites.net/api/v1.0/products", product);

public static const string BackendURL = https://g5store.azurewebsites.net   (annan fil)
Efter en mer permament lösning kan det stå något i stil med return await httpClient.PostJsonAsync<Product>(BackendURL +$"/api/v1.0/products", product);


Andre har gjort en annorlunda lösning än din på sin visa produkter. Denna kanske fungerar